### ISSUE TITLE
Crash on World Generation with GC442(or 443/444), GalaxySpace 1.0.7

### ISSUE THREAD

**Issue by KrazedKopyKat** [BlesseNtumble/GalaxySpace#9]
See crash report at: http://pastebin.com/0xk3MfHG

Upon generating a new world with version 1.0.7 of GalaxySpace, and Galacticraft 442 (or 443 or 444) the game immediately crashes with an Exception Ticking World. I've tried without GalaxySpace, and just Galacticraft 442 (or 443/444) and the crash is not present. Without Biomes o Plenty, the crash is also not present (with GalaxySpace enabled).

**Comment by BlesseNtumble** [comment]
KCauldron?

**Comment by KrazedKopyKat** [comment]
No KCauldron. Using forge 1614, and the aforementioned versions of the mods.

**Comment by KrazedKopyKat** [comment]
I am in the process of narrowing down what is causing the crash. So far I've found if I leave biome configs as their defaults I don't have the issue, but once I change them to the values I used previously (in version 1.0.6.1) it will cause the ticking exception crash. I'm attempting to determine specifically which biome, and at what value, the crash occurs.

**Comment by KrazedKopyKat** [comment]
I've narrowed the issue down to the biome ID of Enceladus. I've changed all other biomes to the values I used in the previous version and there are no issues generating the world. However if I change Enceladus to any value above 222, the crash occurs.

I have attached my configs (that cause the crash) as the first zip on this post. And the second zip are the configs that cause no crashes. I've no idea what would be causing the crash by changing the biome ID to 223 (or above) but that's what I've managed to narrow it down to. It seems biomes o plenty has no effect on the crash either now that I've tested further, so disregard BoP being an issue.

[Crash Configs.zip](https://github.com/BlesseNtumble/GalaxySpace/files/145259/Crash.Configs.zip)
[No Crash Configs.zip](https://github.com/BlesseNtumble/GalaxySpace/files/145260/No.Crash.Configs.zip)
